File: reactive-demo-backend/src/main/java/dev/divezone/demo/reactive/model/Airline.java
package dev.divezone.demo.reactive.model;



import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "AIRLINE")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Airline {

    @Id
    @Column(name = "ID")
    private Integer id;

    @Column(name = "NAME")
    private String name;

    @Column(name = "ALIAS")
    private String alias;

    @Column(name = "IATA", length = 2)
    private String iata;

    @Column(name = "ICAO", length = 3)
    private String icao;

    @Column(name = "CALLSIGN")
    private String callsign;

    @Column(name = "COUNTRY")
    private String country;

    @Column(name = "ACTIVE", length = 1)
    private String active;

    @Column(name = "FAVORITE")
    private boolean favorite = false;

}

File: reactive-demo-backend-kotlin/src/main/kotlin/dev/divezone/demo/reactive/model/Airline.kt
package dev.divezone.demo.reactive.model

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "AIRLINE")
class Airline(
    @Id
    @Column(name = "ID")
    var id: Int? = null,

    @Column(name = "NAME")
    var name: String? = null,

    @Column(name = "ALIAS")
    var alias: String? = null,

    @Column(name = "IATA", length = 2)
    var iata: String? = null,

    @Column(name = "ICAO", length = 3)
    var icao: String? = null,

    @Column(name = "CALLSIGN")
    var callsign: String? = null,

    @Column(name = "COUNTRY")
    var country: String? = null,

    @Column(name = "ACTIVE", length = 1)
    var active: String? = null,

    @Column(name = "FAVORITE")
    var favorite: Boolean = false
)